src/redux/hub/hub.test.ts

```diff
@@ -115,6 +133,24 @@ describe("hub reducer", () => {
         }
       ).error?.name
     ).toEqual("unfortunate error");
+    expect(
```

Comment:
Could you put this into it's own it-block?

Reply:
Done, thanks!